src/engraving/dom/textbase.cpp

```diff
@@ -1893,6 +1893,9 @@ void TextBase::createBlocks(LayoutData* ldata) const
 //---------------------------------------------------------
 bool TextBase::prepareFormat(const String& token, CharFormat& format)
 {
+    static String prevFontFace;
+    static double prevFontSize = 0;
```

Comment:
It doesn't feel great to use `static` here... While it seems unlikely that it will cause problems in the short term, it is annoying when it suddenly turns out that it does cause problems. 
Perhaps the cleanest way might be to create them as (non-static) variables in `TextBase::createBlocks`, and pass them as a reference to `prepareFormat` (via the other `prepareFormat` overload, of course).

Reply:
Shows that I'm a C programmer not a C++ one ;-)

Reply:
Guess I need more help/hints in this :-(

Reply:
I hope you don't mind that I pushed it to this branch as a new commit!

Reply:
No, I don't mind at all. Now I understand what you ment, esp. that you wanted __both__ `prepareFormat()` methods to recieve those variables as references...